wallet/src/de/schildbach/wallet/ui/InputParser.java

```diff
@@ -91,7 +90,7 @@ public StringInputParser(final String input, boolean supportAnypayUrls) {
 
         @Override
         public void parse() {
-            if (input.startsWith(CoinDefinition.coinURIScheme.toUpperCase() + ":-")) {
+            if (input.startsWith("DASH:-")) {
```

Comment:
If we have `SendCoinsActivity.DASH_SCHEME` constant, shouldn't we use it here?

Reply:
Yes, or from something in DashJ.  Will make this change.